packages/react-composites/src/composites/CallComposite/CallScreen.tsx

```diff
@@ -127,7 +126,7 @@ export const CallScreen = (props: CallScreenProps): JSX.Element => {
     <Stack horizontalAlign="center" verticalAlign="center" styles={containerStyles} grow>
       {isInCall(callStatus ?? 'None') ? (
         <>
-          <Stack styles={bannersContainerStyles}>
+          <Stack.Item styles={bannersContainerStyles}>
             <Stack.Item>
```

Comment:
Can a Stack.item be a child of a Stack.item here? (this may need updated to a Stack)